extensions/authentication-jwt/README.md

```diff
@@ -8,6 +8,20 @@ Please note this package has the user service is a reference implementation, not
 recommended for production. You can follow the guide in section
 [Customizing User](#customizing-user) to replace it.
 
+## ⚠️Important Info ⚠️
+
+> The package was originally published as
+> `@loopback/extension-authentication-jwt`. Please use
```

Comment:
Do we really want to mention that? I remember that `@loopback/extension-authentication-jwt` is not usable due to publishing issues. So it should be safe to assume that nobody is using it yet. :-).

Reply:
The concern is that future users may discover the old package name and attempt to use it since it contains the same readme. This addition is to remove confusion of the different package names.

Reply:
We can deprecate the module with a warning there for `@loopback/extension-authentication-jwt`.

Reply:
Sounds good; I've dropped the warning from the readme.